Makefile

```diff
@@ -0,0 +1,13 @@
+GOBIN=$(shell pwd)/bin
```

Comment:
Isn't it a bit excessive to add an entire Makefile just for this? I think instructions could just be part of a `CONTRIBUTING.md` or the `README.md`.

Reply:
Putting this in makefile makes it easier for a user to run the command. In this case a developer can simply check for linting using: `make lint`. Most of the Golang projects use `Makefile` to make it easier to run the commands including `test` and `build`.

Some examples:
* https://github.com/uber-go/zap/blob/master/Makefile
* https://github.com/hashicorp/terraform/blob/main/Makefile